.github/workflows/helm-chart-release.yaml

```diff
@@ -0,0 +1,38 @@
+---
+#
+# This workflow detects Helm chart changes and:
+# 1. Build and push the Helm chart to the GitHub releases
+# 2. Update index.html in GitHub Pages (branch gh-pages) with the latest Helm chart
+#
+name: Release Helm Chart
+
+on:
+  push:
+    branches:
+      - main
+
+jobs:
+  release:
+    # Required to create a release
+    permissions:
+      contents: write
+    runs-on: ubuntu-latest
+    steps:
+      - name: 🏧 Checkout
+        uses: actions/checkout@v4
+        with:
+          fetch-depth: 0
+      - name: 🛠️ Configure Git
+        run: |
+          git config user.name "$GITHUB_ACTOR"
```

Comment:
It's built-in variable https://docs.github.com/en/actions/writing-workflows/choosing-what-your-workflow-does/store-information-in-variables

We need it since `helm/chart-releaser-action` is doing `git commit` and `git push` to `gh-pages` branch, so the commit must have an author.

Reply:
Take a look at this commit - https://github.com/boar-network/helm-chart-test/commit/818271a76a9a992707d8ca554e37d0035c791283
There is `thevops@users.noreply.github.com` because I triggered the workflow.